From: Simon McVittie <smcv@debian.org>
Date: Sun, 22 Dec 2019 21:20:48 +0000
Subject: tests: Always produce verbose output from subprocesses

This helps to debug what is going on if a test fails.

Signed-off-by: Simon McVittie <smcv@debian.org>
---
 tests/test-portals.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tests/test-portals.c b/tests/test-portals.c
index 5eb5a03..425054d 100644
--- a/tests/test-portals.c
+++ b/tests/test-portals.c
@@ -133,7 +133,7 @@ global_setup (void)
 
   backends_executable = g_test_build_filename (G_TEST_BUILT, "test-backends", NULL);
   argv[0] = backends_executable;
-  argv[1] = g_test_verbose () ? "--verbose" : NULL;
+  argv[1] = "--verbose";
   argv[2] = NULL;
 
   g_print ("launching test-backend\n");
@@ -176,7 +176,7 @@ global_setup (void)
   else
     argv[0] = LIBEXECDIR "/xdg-desktop-portal";
 
-  argv[1] = g_test_verbose () ? "--verbose" : NULL;
+  argv[1] = "--verbose";
   argv[2] = NULL;
 
   g_print ("launching xdg-desktop-portal\n");
@@ -217,7 +217,7 @@ global_setup (void)
     argv[0] = LIBEXECDIR "/xdg-permission-store";
 
   argv[1] = "--replace";
-  argv[2] = g_test_verbose () ? "--verbose" : NULL;
+  argv[2] = "--verbose";
   argv[3] = NULL;
 
   g_print ("launching xdg-permission-store");
